spec/support/capybara_helpers.rb

```diff
@@ -2,4 +2,13 @@ module CapybaraHelpers
   def choose_radio_button(label)
     find(:label, text: label).click
   end
+
+  def stub_one_diagnosis_question(answers:, question: 'Dummy question', id: 'location')
```

Comment:
As it's `question` in the singular, could this be called `stub_diagnosis_question` ?

Reply:
This change makes everything a lot tidier :smile:

Reply:
Renamed